pages/1.13/gui/services/index.md

```diff
@@ -35,6 +35,24 @@ By default, all of your services are displayed, sorted by service name. You can
 | Recovering| When a change to the Service has been requested by DC/OS, the instance is killed and a new instance is started. Similar to Deploying, DC/OS is performing the required actions, which have not completed yet. |
 | Stopped | A Service with target instances of 0 and with a running tasks count of 0. This state was previously referred to as “Suspended” in the UI. |
 
+## SDK Service Status
+
+Scheduler tasks based on the [DC/OS SDK](https://mesosphere.github.io/dcos-commons/) are leveraging Mesos native checks in order to provide more detailed information about their status than the above [Service Status](#service-status). These status are specific to SDK based schedulers and their lifecycle.
```

Comment:
isn't the english plural "statuses"?

Reply:
In Oxford English it's _status_; I guess the US is a little more flexible ;)
I can imagine it to more common in the US to write _statuses_, and it might be confusing to read the correct plural of a latin word. I'll leave it up to our Tech Writer's Guild™️ to make a call.